pkgs/development/libraries/libantlr3cpp/default.nix

```diff
@@ -0,0 +1,25 @@
+{stdenv, fetchFromGitHub}:
+
+stdenv.mkDerivation {
+  name = "libantlr3cpp-3.5";
+  src = fetchFromGitHub {
+    owner = "antlr";
+    repo = "antlr3";
+    rev = "5c2a916a10139cdb5c7c8851ee592ed9c3b3d4ff";
+    sha256 = "1i0w2v9prrmczlwkfijfp4zfqfgrss90a7yk2hg3y0gkg2s4abbk";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/include
+    cp runtime/Cpp/include/* $out/include/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "C++ runtime libraries of ANTLR v3";
```

Comment:
Given the `installPhase` you have provided this sounds misleading. What is the difference between this package and `antlr`?

Reply:
Good question! For some reason, even though the `antlr` package provides libraries, these are currently at version `2.7.7` which is incompatible with `3.x`, and the existing `antlr3` package does not include any libraries.

Maybe I'm missing something though?

Reply:
Oh I see... If I had to guess and if I trusted my memory on the issue (which I definitely do not :smile:) I would wonder if when`antlr3` was packaged for nixpkgs back in 2016 maybe the c++ support in `antlr3` wasn't that good so it was just skipped... :man_shrugging:

Maybe we should just amend the existing package?

Reply:
Okay good point, I'll look into that!

Reply:
Amended the existing package instead! Let me know if that looks okay, please.